### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by lisa005**
Edits for overall tone, grammar, and conciseness.

### CODE REVIEW

File: exampleSite/content/docs/faq.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ menu:
     name: FAQ
 ---
```
Comment (Vimux) [inline exampleSite/content/docs/faq.md:14]: I use 120 as a maximum line length for Mainroad markdown content (exceptions: long links and code blocks, etc.). My fault. I need to mention this in the guidelines or set up automatic validation in the future.

I'm not sure what editing tools you use, so let me know if you can fix this on your end?

**Review by Vimux — CHANGES_REQUESTED** [review]

File: exampleSite/content/docs/faq.md (hunk 1 above)
Comment (lisa005) [reply exampleSite/content/docs/faq.md:14]: Understood; I tried to go through and format for length.

File: exampleSite/content/docs/faq.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ The answers have been categorized into two groups:
...
-### Do I need to have some prior experience before proceeding with the Mainroad theme?
+### Do I need to have prior experience before proceeding with the Mainroad theme?
 
-**Yes.** We expect that you already have prior experience with Hugo, at least.
-[Our docs section]({{< ref "/docs/_index.md" >}} "Mainroad theme documentation") is for intermediate users and
-developers, not novice users with zero or minimal experience. Still, all documentation pages would be helpful even if
-you don't have such experience. Just don't expect all your typical questions answered here.
+**Yes.** You'll need to be familiar with Hugo before proceeding.
+[Our docs section]({{< ref "/docs/_index.md" >}} "Mainroad theme documentation") is intended for intermediate to 
+advanced users anddevelopers. Our documentation may still be helpful to users with minimal experience, but are not
```
Comment (Vimux) [inline exampleSite/content/docs/faq.md:31]:
```suggestion
advanced users and developers. Our documentation may still be helpful to users with minimal experience, but are not 
```

File: exampleSite/content/docs/faq.md (hunk 1 above)
Comment (Vimux) [inline exampleSite/content/docs/faq.md:14]: One more thing: trim single spaces at the end of lines. See [285#pullrequestreview-893651388](https://github.com/Vimux/Mainroad/pull/285#pullrequestreview-893651388) for more info.

**Review by Vimux — CHANGES_REQUESTED** [review]

File: exampleSite/content/docs/faq.md (hunk 1 above)
Comment (lisa005) [reply exampleSite/content/docs/faq.md:14]: Will do!

**Review by Vimux — APPROVED** [review]

**Comment by Vimux** [comment]
Really appreciate your help, thanks.

**Pull request merged**